modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleaner.java

```diff
@@ -66,9 +66,12 @@ protected void activate(BundleContext bundleContext)
 			bundleContext.createFilter(
 				StringBundler.concat(
 					"(&(", Constants.OBJECTCLASS, "=", Store.class.getName(),
-					")(store.type=", S3Store.class.getName(), "))")),
+					")(store.type=", S3Store.class.getName(),
+					")(current.store=true))")),
```

Comment:
@yuhai @dantewang 
Is this change related to scheduler? If not, let's create a new ticket and make the change.

Also, is this a bug? if yes, please provide more info in the comment and steps to reproduce it in the new ticket.

Thanks.
Tina